sentinel-core/slot/statistic_slot.cc

```diff
@@ -39,6 +39,8 @@ TokenResultSharedPtr StatisticSlot::OnPass(const EntrySharedPtr& entry,
                                            const Stat::NodeSharedPtr& node,
                                            int count, int) {
   this->RecordPassFor(node, count);
+  this->RecordPassFor(Stat::ResourceNodeStorage::GetInstance().entry_node(),
```

Comment:
This can be recorded only if `EntryType == IN` (entryType can be retrieved from ResourceWrapper)